policyengine_us/variables/gov/states/mi/tax/income/credits/homestead_property_tax_credit/mi_homestead_eligible.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class mi_homestead_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Michigan homestead property tax credit"
+    definition_period = YEAR
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-508",
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/MI-1040CR.pdf#page=2",
+        "https://www.legislature.mi.gov/Publications/TaxpayerGuide.pdf#page=16",
+    )
+    defined_for = StateCode.MI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.credits.homestead_property_tax_credit
+
+        property_value_eligible = (
+            add(tax_unit, period, ["assessed_property_value"])
+            <= p.property_value_limit
+        )
+
+        exceed_amount_eligible = (
+            tax_unit("mi_homestead_property_tax_credit_non_refundable", period)
```

Comment:
do you only get the credit if the non-refundable amount is over 0? or is it some form of excess?

Reply:
https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/MI-1040CR.pdf#page=2
> 35. Subtract line 34 from line 13 and enter the amount here. If line 34 is greater than line 13, enter “0” and STOP; you are not eligible for this credit.